### PR TITLE
docs(README): correct webpack example link

### LINKED ISSUE #3897: README: Invalid link for webpack under example projects
## Bug Report

Invalid link for webpack under example projects in README.md

### Steps
open README
go to Example Projects.
click on webpack.
See Error

## Expected Result

It should be redirected ti correct url.

### Actual Result
Screenshot from 2020-02-10 21-06-04

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3897

### CODE REVIEW

**Review by tkovs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -85,13 +85,7 @@ This is a listing of example projects and guides that will help you integrate Se
 
 <details>
   <summary>Show projects</summary>
-
-  ### [webpack][28]
-  See our webpack 3 example project [here][28] (includes theming).
-   
-  ### [SUIcrux][102]
```
Comment (layershifter) [inline README.md:92]: image

The project is not maintained anymore.

File: README.md [hunk 2]
```diff
@@ -85,13 +85,7 @@ This is a listing of example projects and guides that will help you integrate Se
 
 <details>
   <summary>Show projects</summary>
-
-  ### [webpack][28]
-  See our webpack 3 example project [here][28] (includes theming).
```
Comment (layershifter) [inline README.md:90]: Actually this link should be removed at all as these examples were removed at all (#3729).

**Review by layershifter — APPROVED** [review]

**Comment by snitin315** [comment]
@layershifter How can I become member of Semantic-UI org on GitHub?

**Comment by layershifter** [comment]
@snitin315 we don't have a policy around the membership and actually I don't control it 🤔

Others did significant impact on SUIR at some point, so contributions are the best option to become a member _I_ would say...

**Pull request merged**